content/en/cloud_cost_management/tags/_index.md

```diff
@@ -0,0 +1,126 @@
+| Google Cloud | Google Billing Export - Project Labels| User-defined labels on a project in Google Cloud, found in the project.labels column in the billing export |
+| Google Cloud | Google Billing Export - System Resource Labels | System-generated labels on a resource in Google Cloud, found in the system_labels column in the billing export |
+| Google Cloud | Google Billing Export - User Resource Labels | User-defined labels on a cloud resource in Google Cloud, found in the labels column in the billing export |
+| Google Cloud | Google Billing Export - User Resource Tags | User-defined tags on a cloud resource in Google Cloud, found in the tags column in the billing export. The `goog-originating-sku-description` tag is added leveraging Google's SKU APis, to provide more granular SKU details for commitment line items. |
+| Google Cloud | GKE Pod | User-defined labels found on pods running in Google Kubernetes Engine | 
+| Oracle Cloud | OCI Cost Export - User Resource Tags | User-defined tags on a cloud resource in Oracle Cloud Infrastructure, from the Tags column in the OCI FOCUS cost export |
+| Datadog | Datadog Usage Attribution | User-defined tags for Usage Attribution in Datadog Plan and Usage |
+| Custom Costs | Cost File Tags | User-defined tags for every provider, found on [cost files][9] uploaded to Cloud Cost Management |
+
+## How tags are normalized
+
+Tag keys and/or values may look slightly different in Cloud Cost Management compared to the providers or other parts of Datadog because of tag normalization. 
+
+Cloud Cost Management normalizes tag **keys** largely the same as Datadog Metrics:
+- Drop leading characters that are not letters
+- Lower-case all characters
+- Replace special characters and spaces with single underscore
+- Remove any trailing underscore
+- Reduce contiguous underscores to a single underscore
+- Tag keys up to 5000 characters are supported, and any characters up until a letter are dropped so that tag keys start with letters (differs from Datadog Metrics)
+
+Cloud Cost Management normalizes tag **values** as well, while maintaining human readable tag values for cost reporting, with the following logic:
+- Convert any consecutive whitespace to a single space
+- Keep all letters, marks, numbers, punctuation, and symbols
+- Replace any other characters with an underscore `_`
+- Tag values up to 5000 characters are supported
+
+## How tags are prioritized
+
+It's possible for a cost data row to have multiple values, if tag values from 2+ sources are combined, and one is not prioritized over the other.
+
+To resolve conflicts and mitigate this, Cloud Cost Management will replace existing tags instead of adding duplicates using the most specific source for each tag key. For example, a Kubernetes Pod tag `team:shopist` would take precedence and replace an Kubernetes node tag `team:compute`.
```

Comment:
```suggestion
To resolve conflicts and mitigate this, Cloud Cost Management will replace existing tags instead of adding duplicates using the most specific source for each tag key. For example, a Kubernetes Pod tag `team:shopist` would take precedence and replace a Kubernetes node tag `team:compute`.
```